### ISSUE TITLE
[NextJS] Deploying Custom Headers

### ISSUE THREAD

**Issue by Estreuselito** [ionos-deploy-now/ionos-deploy-now#68]
Hi there,

I am currently deploying my first Next.js website using Deploy Now. I wanted to improve the security of my deployed webpage by adding custom headers to my Next.js application. I did this by adding a `headers.js` file with the custom headers and a modification in `next.config.js`. When running my webpage locally with `npm run dev` the headers are displayed, however, when I deploy my website using the Deploy Now service, the headers are not deployed to my webpage.

**TL;DR**

For Security:
- Added `header.js` to `next.js` application
- Modified `next.config.js` accordingly
- Headers show up in localhost
- Header **do not** show up on my domain

Thanks for taking care of this issue.

**Comment by druvio** [comment]
Hello,

thank you for reporting your issue.
A ticket has been opened to address it as soon as possible.

In the meantime, in order to investigate the issue in detail, could you please send us a copy of the deployment logs at deploynow-support@ionos.com?

**Comment by Estreuselito** [comment]
Hi @druvio,
I just sent a copy of the logs. :). Hope we can resolve this soon.
